packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -14,14 +14,18 @@ import "../../oracle/interfaces/StoreInterface.sol";
 import "../../oracle/interfaces/FinderInterface.sol";
 import "../../oracle/implementation/Constants.sol";
 
-import "../funding-rate-store/interfaces/FundingRateStoreInterface.sol";
+// TODO: point this at an interface instead.
```

Comment:
Are you gonna do this in a follow up PR?

Reply:
Yeah, I think we should leave this for a follow-up since it doesn't change any core logic.